### PR TITLE
Myles/gap status

### PR DESCRIPTION
**Pull request by mylesdomingo**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

1. Added babelrc support for multiple imports
2. Created FarmCertificationStepper for Farm Profile
3. Fixed CSS bugs for FarmProfile + update to match Figma
4. Added MUI Chip Component
5. Separated Farm Profile into multiple child components
6. Integrated Material UI Icons + colors

### How to Review
1. Go to Farm Profile and see it matches Figma doc

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
1. Airtable Data to be pulled in and parsed.
2. Add bold weight headers to App.css
3. Validate undefined props for Farm Profile
4. Adapt Stepper for smaller screens (<1440px width)
5. Move FarmProfile CSS to MUI styles
6. Finalize styling for chip component

### Screenshots
Before:
image

After:
image

CC: @phoebeli23

### CODE REVIEW

File: src/components/Stepper.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    whiteSpace: 'pre-wrap'
+  },
+  date: {
+    margin: 4,
+    color: 'var(--ks-medium-dark-grey)'
+  }
+});
+
+const useIconStyles = makeStyles({
+  error: {
```
Comment (mylesdomingo) [inline src/components/Stepper.js]: todo: move icon styles to new file

File: src/components/Stepper.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    error: [classes.error, <ErrorOutline fontSize="large" />],
+    activeCompleted: [classes.active, <CheckCircle fontSize="large" />],
+    active: [classes.active, <Cancel fontSize="large" />],
+    completed: [classes.inactive, <CheckCircle fontSize="large" />],
+    incompleted: [classes.inactive, <RadioButtonUnchecked fontSize="large" />]
+  };
+
+  if (error) state = icons.error;
+  else if (active && completed) state = icons.activeCompleted;
+  else if (active) state = icons.active;
```
Comment (mylesdomingo) [inline src/components/Stepper.js]: switch with switch-case statement

File: src/components/Stepper.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    'Food Safety Plan Complete',
+    'Risk Assessment',
+    'Mock Recall Complete',
+    'Internal Audit Complete (1)',
+    'Internal Audit Complete (2)',
+    'Group GAP Certified!'
+  ];
+}
+
+export default function HorizontalLabelPositionBelowStepper() {
```
Comment (mylesdomingo) [inline src/components/Stepper.js]: todo: replace with class component

Comment (phoebeli23) [reply src/components/Stepper.js:95]: was just about to ask about this but i see this comment 👍

File: src/components/Stepper.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  const error = [6, 7];
+
+  return {
+    completed: completed.includes(index),
+    active: active.includes(index),
+    error: error.includes(index)
+  };
+}
+
+function getSteps() {
```
Comment (phoebeli23) [inline src/components/Stepper.js]: thoughts on putting getSteps() in constants.js, since it will be needed in other pages (like the dashboard)?

**Review by phoebeli23 — COMMENTED** [review]
- Beautiful modularization (breaking down FarmProfile into FarmGraphsTable and Stepper, adding StatusChip to display GAP status in FarmContactCard)
- Good catch with the Navbar
- Babelrc will be useful for other devs
- Looking forward to the next updates

File: src/components/Stepper.js (hunk 4 above)
Comment (mylesdomingo) [reply src/components/Stepper.js]: will handle in a future PR as needed.

**Review by phoebeli23 — APPROVED** [review]
Looks good to me!

**Pull request merged**